src/components/VideoDetails/VideoDetails.test.jsx

```diff
@@ -1,21 +1,29 @@
 import React from 'react';
-import { render, screen } from '@testing-library/react';
+import { act, render, screen } from '@testing-library/react';
 import { createMemoryHistory } from 'history';
 import { Router } from 'react-router-dom';
+import { setupServer } from 'msw/node';
 import VideoDetails from './VideoDetails.component';
+import { GlobalContextProvider } from '../../providers/GlobalContext/GlobalContext';
+import { handlers } from '../../mocks/handlers';
```

Comment:
This is overkill for the test cases. Also ads a lot of boilerplate to your project which I think overcomplicates your testing. You should be good if you just mock `axios` (or `fetch`) calls. Let me know if you have any questions about it.